pallets/pools/src/lib.rs

```diff
@@ -520,42 +572,46 @@ pub mod pallet {
 
 			if let Ok(order) = Order::<T>::try_get(&tranche, &who) {
 				ensure!(
-					order.supply.saturating_add(order.redeem) == Zero::zero()
+					order.invest.saturating_add(order.redeem) == Zero::zero()
 						|| order.epoch == epoch,
 					Error::<T>::CollectRequired
 				)
 			}
```

Comment:
I've created a separate ticket for this: https://github.com/centrifuge/centrifuge-chain/issues/591

Reply:
`try_get` bypasses the `Query` type and always returns a `Result`. You can see the implementation [here](https://github.com/paritytech/substrate/blob/master/frame/support/src/storage/generator/double_map.rs#L138-L152), though it's not exactly the most clear and is buried in template magic.

It may still be more semantically correct for us to use `OptionQuery` with `get` for things that we expect to be absent.